src/System.Private.Xml/tests/Xslt/XslCompiledTransformApi/XsltArgumentList.cs

```diff
@@ -3019,27 +3019,19 @@ public void AddExtObject24()
         [InlineData(XslInputType.Navigator, ReaderType.XmlValidatingReader, OutputType.Writer, NavType.XPathDocument)]
         [InlineData(XslInputType.Navigator, ReaderType.XmlValidatingReader, OutputType.TextWriter, NavType.XPathDocument)]
         [Theory]
-        public void AddExtObject25(XslInputType xslInputType, ReaderType readerType, OutputType outputType, NavType navType)
+        public void TC_ExtensionObj_Function_Mismatch_IncorrectCasing(XslInputType xslInputType, ReaderType readerType, OutputType outputType, NavType navType)
         {
             MyObject obj = new MyObject(25, _output);
             m_xsltArg = new XsltArgumentList();
 
             m_xsltArg.AddExtensionObject(szDefaultNS, obj);
-
-            if (LoadXSL("MyObject_CaseSensitive.xsl", xslInputType, readerType) == 1)
+            LoadXSL("MyObject_CaseSensitive.xsl", xslInputType, readerType);
+            var e = Assert.ThrowsAny<XsltException>(() => Transform_ArgList("fruits.xml", outputType, navType));
+            if (!PlatformDetection.IsFullFramework)
```

Comment:
I tried with the information shown in the output of the failure run but the check failed with that too. To be fair I didn't debug `CheckExpectedError` to understand exactly why.